assets/admin/editor-wizard/wizard.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Wizard component.
+ *
+ * @param {Object}   props
+ * @param {Array}    props.steps        Array with the steps that will be rendered.
+ * @param {Function} props.onCompletion Callback to call when wizard is completed.
+ */
+const Wizard = ( { steps, onCompletion } ) => {
+	const [ currentStepNumber, setCurrentStepNumber ] = useState( 0 );
+	const [ data, setData ] = useState( {} );
+
+	const CurrentStep = steps[ currentStepNumber ];
+
+	const goToNextStep = () => {
+		if ( currentStepNumber + 1 < steps.length ) {
+			setCurrentStepNumber( currentStepNumber + 1 );
```

Comment:
Probably not critical here, but just a tip for the future: Always when a new state depends on the previous one, it's nice to set it with a callback getting the previous state, like this:

```
setCurrentStepNumber( prev => prev + 1 );
```

It helps to avoid async weird issues when using the setter function quickly.

Reply:
Cool! I had to do this in the Standalone Questions too but I forgot. Thanks for refreshing me about it!

Edit: I was going to change it but I think I will keep it as it is. I want the "if" condition and the set be consistent. I think using `prev => prev + 1` could have some side-effect regarding the limits. What do you think?

Reply:
Makes sense! And this case won't be a big deal because it's something very unlike to call quickly in a row.

Reply:
> The main issue with this approach would be the call to the onCompletion callback, which would be definitely weird to call here. Maybe it would make sense to call it in a useEffect hook or similar, tho, based on this same condition. 🤔

This made sense to me, so I was implementing it when I realised this would imply setting a "currentStep" out of range, which complicates a little the logic. So I will keep it as it is. I think the current risk would be that some "clicks" might be ignored, and that's only for REALLY fast clicks. So I think we are safe :)